tox.ini

```diff
@@ -1,9 +1,10 @@
-; a generative tox configuration, see: https://testrun.org/tox/latest/config.html#generative-envlist
+; a generative tox configuration, see: https://tox.readthedocs.io/en/latest/config.html#generative-envlist
 
 [tox]
 envlist =
     check
-    {py27,py34,py35,py36,py37,pypy,pypy3}-{t310,t43,t44,t45}-{c45}
+    {py27,py34,py35,py36,py37,pypy,pypy3}-{t310-x27,t43-x27,t44-x28,t45-x28}-{c45}
+    py37-t310-x22-c45
```

Comment:
Add a comment that this is the minimal supported config?  (if that's the case)

Reply:
It's not minimum, just an arbitrary xdist 1.22.0 test environ

Reply:
Ok. Then it should use the minimum versions maybe?

Reply:
Like py27? That's going out of support sooner than xdist==1.22.0

Reply:
I think this is now clear with the longer toxenv factor names